src/compat/eme/custom_media_keys/index.ts

```diff
@@ -100,13 +102,8 @@ if (isNode ||
     b : MediaKeySystemConfiguration[]
   ) : Observable<MediaKeySystemAccess> =>
     castToObservable(navigator.requestMediaKeySystemAccess(a, b));
-} else {
-  /* eslint-disable @typescript-eslint/no-unsafe-call */
-  /* eslint-disable @typescript-eslint/no-unsafe-member-access */
+} else if ((MediaKeys_ as any).isTypeSupported !== undefined) {
```

Comment:
I'm not sure here that the any casting is necessary (But not imputed to this PR)

Reply:
Done